proto/lighthouse-result.proto

```diff
@@ -138,10 +149,49 @@ message LighthouseResult {
 
     // List of the categories that were run, empty if all were run
     // nullable list of strings
+    // Note: this should have been google.protobuf.ListValue, but too late now.
```

Comment:
What impact does this have on us? (same with onlyAudits I assume?

Reply:
Little. The python / go proto serializers handle both just fine, given a list of strings. Technically LR users can pass anything to `only_categories` (like a string or a struct), and it would pass the serialization with flying colors, but ultimately fail the request (or at least, the setting would get ignored. i have not tried it). But the same is true for ListValue (pass a list of int32s, no bueno). so it's already an existing shortcoming.